docs/src/pages/ui/theme/alternativeStyling/index.page.mdx

```diff
@@ -0,0 +1,501 @@
+
+##### CSS Root Variable Overrides
+
+Amplify theming provides many CSS variables to customize the look and feel of the entire application. These CSS variables can be used at a root level to override all instances of component styling.
+
+```jsx
+/** @jsxImportSource @emotion/react */
+import { jsx, css } from '@emotion/react';
+import { Button } from '@aws-amplify/ui-react';
+
+const styles = {
+  color: 'blue',
+};
+
+<Button css={css(styles)}>Button</Button>;
+```
+
+```CSS
+/* External Style Sheet */
+:root {
+  --amplify-components-button-color: red;
+}
+```
+
+In the example above, the button color will be set to `blue` because the Emotion styling will take precedence over the Amplify CSS variable.
+
+</Block>
+
+<Block name="Aphrodite">
+[Aphrodite](https://github.com/Khan/aphrodite) is a Framework-agnostic CSS-in-JS with support for server-side rendering, browser prefixing, and minimum CSS generation.
+
+#### Usage
+
+```jsx
+import { StyleSheet, css } from 'aphrodite';
+import { Text } from '@aws-amplify/ui-react';
+
+const styles = StyleSheet.create({
+  blue: {
+    color: 'blue',
+  },
+});
+
+//use the Aphrodite css function and the className prop
+<Text className={css(styles.blue)}>This is an aphrodite Text component</Text>;
+```
+
+#### Interactions
+
+There are various ways to customize an Amplify component and Aphrodite will interact with these customizations in the following ways:
+
+##### Amplify Styling Props
+
+These are styling props that can be used directly on an Amplify component which will affect a single style property. Examples of styling props are `color` or `fontWeight`. Aphrodite will take precedence over these Amplify styling props.
+
+```jsx
+import { StyleSheet, css } from 'aphrodite';
+import { Text } from '@aws-amplify/ui-react';
+
+const styles = StyleSheet.create({
+  blue: {
+    color: 'blue',
+  },
+});
+
+<Text className={css(styles.blue)} color="red">
+  This is an aphrodite Text component
+</Text>;
+```
+
+In the example above, the color of the view will be set to `blue` because of the Aphrodite styling will take precedence. [Here]() is a list of all Amplify styling props.
+
+##### Amplify variation props
+
+These are Amplify props that are added to certain Amplify components to change the look and/or behavior of the Amplify component. Examples of these variation props are `size` or `variation`.
+
+```jsx
+import { StyleSheet, css } from 'aphrodite';
+import { Button } from '@aws-amplify/ui-react';
+
+const styles = StyleSheet.create({
+  blue: {
+    color: 'blue',
+  },
+});
+
+<Button className={css(styles.blue)} variation="primary">
+  Primary Button
+</Button>;
+```
+
+In the example above, the color of the button will be set to `blue`, because the Aphrodite styling will take precedence over the Amplify variation styling. A list of Amplify variations can be found on [here]().
```

Comment:
Same here, all other libraries this would be 'white', is that expected?

Reply:
yes by default aphrodite adds !important to all of their styling, there is a config option to turn this off but I just have the default behavior covered here.